domains/vice.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "antonvice",
+        "email": "anton96vice@gmail.com",
+    },
+    "records": {
+        "URL": "https://antonvice.github.io/my-blog/"
+    },
+    "proxied": false
```

its already the default
```suggestion
    }
```